.ci/apply_settings.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ "$check_haddock" != @(True|False) ]]; then
+  echo "check_haddock: Expected True or False, got \"$check_haddock\"" >&2
+  exit 1
+fi
+sed <.ci/cabal.project.local.in >cabal.project.local "
+    s/__CHECK_HADDOCK__/$check_haddock/"
```

Comment:
Why not `sed -i`?

Reply:
But then you'd still have to copy the file, right?

If I try to use `--in-place`, I end up with something like:

```
sed .ci/cabal.project.local.in -i -e "
    s/__CHECK_HADDOCK__/$check_haddock/"
cp .ci/cabal.project.local.in cabal.project.local
```